templates/build/small-h-card-detail.build.hbs

```diff
@@ -41,22 +41,24 @@
         <!-- Image template -->
         \{{#ifCond type '==' 'image'}}
           \{{#if content}}
-            \{{#if labelEnabled}}
-              <div class="small-h-card-list-detail-label">\{{label}}</div>
-            \{{/if}}
-            \{{#if (isSingle contentArray)}}
-              <div class="small-h-card-list-detail-body-text">
-                <img src="\{{validateImage content}}">
-              </div>
-            \{{else}}
-              <div class="multiple-images-holder" data-detail-entry-id="\{{id}}">
-                \{{#each contentArray}}
-                  <div class="multiple-images-item focus-outline" tabindex="0">
-                    <div class="image" style="background-image: url('\{{validateImage this}}')"></div>
-                  </div>
-                \{{/each}}
-              </div>
-            \{{/if}}
+            <div data-detail-entry-id="\{{id}}">
```

Comment:
Perhaps this isn't needed at all?

https://github.com/Fliplet/fliplet-widget-dynamic-lists/blob/ebf7570206acfae7546d75984919e2d5939bec6a/templates/build/small-h-card-detail.build.hbs#L7 shows that the detail view overlay always has a container that highlights the entry ID. You can just access that through `.parents('[data-entry-id]')`. It'll simplify a lot of the changes you've made here.

Reply:
@tonytlwu they are different, for example.
This is `data-detail-entry-id` isn't equal to the `data-entry-id`
![image](https://user-images.githubusercontent.com/53430352/115398151-8674f680-a1ef-11eb-99dd-8dcb9018c7fd.png)

`data-detail-entry-id` is equal to the `dynamicDataObj.id` what we use at this [place](https://github.com/Fliplet/fliplet-widget-dynamic-lists/blob/master/js/layout-javascript/small-card-code.js#L1957)